src/olympia/versions/migrations/0031_rename_nomination_version_due_date.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.16 on 2023-01-07 00:47
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('versions', '0030_auto_20221122_1312'),
+    ]
+
+    operations = [
+        migrations.RenameField(
```

Comment:
This will break all `Version` queries during the push - I'm not sure how fast the migrations will be, so that seems problematic. Since we're rewriting the values anyway, we should probably add a new field instead, doing the usual dance of removing the old field from models, then doing a SQL migration to remove the column next week.

Reply:
https://github.com/mozilla/addons-server/pull/20151/commits/d1fcaa1b583df3b4df866ef50a6d458cc533c9f4